Translation corrections (#81)
07th-mod/higurashi-console-arcs: Update/some_004.txt

//rそして、意を決したように視線を上げると、藤堂くんは振り絞るように、言葉をつむぎだしていった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、意を決したように視線を上げると、藤堂くんは振り絞るように、言葉をつむぎだしていった。",

> 			NULL, "And then, he looked up at me as if he'd resolved myself, then mustered his courage and carefully chose his words.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And then, he looked up at me as if he'd resolved himself, then mustered his courage and carefully chose his words.", Line_Normal);
```

//rぶるぶると小刻みに震え続ける千紗登ちゃん。震えを押し殺そうと、更にギュッと両手を握り締める。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ぶるぶると小刻みに震え続ける千紗登ちゃん。震えを押し殺そうと、更にギュッと両手を握り締める。",
			NULL, "Chisato-chan's shoulders continued to tremble more and more. She clenched her fists even tighter to ease the shaking.", Line_Normal);
	ClearMessage();

//r血色の良い千紗登ちゃんの手のひらに爪がぐいぐいぐいぐい食い込んでいくのが見えて、…私の全身の肌を、あわ立つように悪寒が走り抜けていった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "血色の良い千紗登ちゃんの手のひらに爪がぐいぐいぐいぐい食い込んでいくのが見えて、…私の全身の肌を、あわ立つように悪寒が走り抜けていった。",

> 			NULL, "I could see Chisato-chan's fingernails digging into her palms deep enough to bleed... A shiver ran down my spine as my hairs stood on end.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I could see Chisato-chan's fingernails digging into her palms deep enough to draw blood... A shiver ran down my spine as my hairs stood on end.", Line_Normal);
```

//rでも、言えない。…千紗登ちゃんの異様な威圧で、口に出すことができなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも、言えない。…千紗登ちゃんの異様な威圧で、口に出すことができなかった。",

> 			NULL, "But I couldn't say it. I couldn't bring myself to speak in the face of Chisato-chan's daunting aura.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But I couldn't say it. I couldn't bring myself to speak up when faced with Chisato-chan's daunting aura.", Line_Normal);
```

//r千紗登ちゃんは上半身を仰け反らせると、けたたましいほどの笑い声を響かせる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "千紗登ちゃんは上半身を仰け反らせると、けたたましいほどの笑い声を響かせる。",
			NULL, "Chisato-chan bent her upper body back with a shrill sound of laughter.", Line_Normal);
	ClearMessage();

//r引きつったような、千紗登ちゃんの歪んだ笑顔。…わけもわからず私は足がすくんで、…震える手を机の上につき、倒れそうな身体を支え込んだ……。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "引きつったような、千紗登ちゃんの歪んだ笑顔。…わけもわからず私は足がすくんで、…震える手を机の上につき、倒れそうな身体を支え込んだ……。",

> 			NULL, "As if she was spasming, Chisato-chan's face warped into a smile. ...My legs were frozen with confusion... I put a trembling hand on the desk to keep myself from collapsing......", Line_Normal);

Reviewer: ```suggestion
			NULL, "As if she was spasming, Chisato-chan's face warped into a smile. ...My legs were frozen in my confusion... I put a trembling hand on the desk to keep myself from collapsing......", Line_Normal);
```
